modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -81,74 +91,76 @@
-								<div class="font-weight-semi-bold h2 mt-1">
-									${product.name.en_US}
-								</div>
-
 								<#if productSpecifications?has_content>
 									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
 
 									<#list productPriceModels as productPriceModel>
-										<div class="color-neutral-3 font-size-paragraph-small mt-1">
+										<div class="developer-name font-size-paragraph-small">
 											${productPriceModel.value.en_US}
 										</div>
 									</#list>
 								</#if>
+
+								<div class="font-weight-semi-bold h2 mt-1">
+									${product.name.en_US}
+								</div>
 							</div>
 						</div>
 
 						<div class="d-flex flex-column font-size-paragraph-small h-100 justify-content-between p-2">
-							<div>
-								<div class="font-weight-normal mb-2">
-									${productDescription}
-								</div>
+							<div class="font-weight-normal mb-2">
+								${productDescription}
+							</div>
 
-								<#if productCategories?has_content>
-									<div class="align-center d-flex labels">
-										<#list productCategories as category>
-											<#if category.vocabulary=='marketplace solution category'>
-												<div class="border-radius-small category-label font-size-paragraph-small font-weight-semi-bold px-1">
-													${category.name}
-												</div>
-												<#break>
-											</#if>
-										</#list>
-
-										<#if (productCategories?size > 1)>
-											<div class="category-label-remainder pl-2 position-relative text-primary">
-												+${productCategories?size - 1}
-												<div class="category-names font-size-paragraph-base p-4 position-absolute rounded text-white">
-													<#list productCategories as category>
-														<#if !category?is_first && category.vocabulary == 'marketplace solution category'>
-															${category.name}
-															<#sep>, </#sep>
-														</#if>
-													</#list>
-												</div>
+							<#if productCategories?has_content>
+								<#assign solutionCategories = productCategories?filter(category -> category.vocabulary == 'marketplace solution category') />
+
+								<div class="align-items-center d-flex labels">
+									<#list solutionCategories as category>
+										<div class="border-radius-small category-label font-size-paragraph-small font-weight-semi-bold px-1 rounded">
+											${category.name}
+										</div>
+										<#break>
+									</#list>
+
+									<#assign productCategoriesItems = solutionCategories?map(category -> category.name) />
+
+									<#if (productCategoriesItems?size > 1)>
+										<div class="category-label-remainder pl-2 position-relative text-primary">
+											+${productCategoriesItems?size - 1}
+											<div class="category-names font-size-paragraph-base p-4 position-absolute rounded text-white">
+												<#list productCategoriesItems as category>
+													<#if category_index != 0>
```

Comment:
I was not able to find this variable. Could you explain me?

Reply:
The `category_index` is an implicit variable that contains the index of the current item in the list. This causes it to skip the first item in the list.